packages/expo-screen-orientation/src/ScreenOrientation.types.ts

```diff
@@ -0,0 +1,62 @@
+export enum Orientation {
+    UNKNOWN = 'UNKNOWN',
```

Comment:
We should probably use integers in all those enums instead of strings. cc @ide @sjchmiela